indexing-service/src/main/java/io/druid/indexing/common/task/AbstractTask.java

```diff
@@ -129,6 +129,19 @@ public String getClasspathPrefix()
   }
 
   @Override
+  public boolean canRestore()
+  {
+    return false;
+  }
+
+  @Override
+  public void stopGracefully()
+  {
+    // Should not be called when canRestore = false.
```

Comment:
do u want to explicitly check for the condition and print this message when canRestore is false?